uiowa/uiowa
# Admissions Student Profiles Block Updates

Resolves https://github.com/uiowa/uiowa/issues/4689

## Stories

As a site editor, I want to display a random set of student profiles but might not always have enough results for a nice looking grid of three.

As a site editor, sometimes I want a list of student profiles showing their name and major but still want to filter on student profile data.

As a visitor, the grid/hide details option is presented to me stays as a grid on mobile.

## To Test

- Sync admissions.uiowa.edu locally (no import errors)
- Create a new student profile and make sure there are no issue with the existing grid block in the right sidebar.
- Visit one of the random grid items to make sure the link works and again, there are no issues with the grid block on an existing piece of content.
- Go to a basic page and add a student profile grid block changing the display configuration from list to grid and trying the column options as well as the hide details option. It should be a decent result except for maybe a list/hide details combination.
- Check node/491 and node/4896 (existing blocks on pages) to see that they are updated and similar looking (gray bg, grid, three column, hide details)
